nautobot/extras/filters/__init__.py

```diff
@@ -610,6 +623,7 @@ class ExportTemplateFilterSet(BaseFilterSet):
         },
     )
     owner_content_type = ContentTypeFilter()
+    content_type = ContentTypeFilter(accept_pk=True)
```

Comment:
For backwards compatibility with this `content_type` filter I had to use this approach.
`nautobot.extras.tests.test_filters.ExportTemplateTestCase.test_content_type` test failed since it was just relying on just `pk`

Reply:
Any reason why `accept_pk` should even be an option on the filter? Should it just *always* support both patterns?